unblob/handlers/filesystem/romfs.py

```diff
@@ -64,11 +62,11 @@ def get_string(file: File) -> bytes:
     return filename.rstrip(b"\x00")
 
 
-class FileHeader(object):
+class FileHeader:
     addr: int
     next_filehdr: int
     spec_info: int
-    type: FS_TYPE
+    type_: FSType
```

Comment:
I very much like [flake8-builtins](https://beta.ruff.rs/docs/rules/#flake8-builtins-a) - I don't like shadowing variables.

I think a fine(er) replacement for the occurred problems in this commit would be to rename the variables  in a more .... meaningful ... name than just adding a `_` suffix.

If you have the energy / creativity to do so that would be better ... otherwise, it is okay, I can sacrifice those names for the greater good :sweat_smile:

Reply:
Would using `fs_type` as variable name be acceptable to you @kukovecz ?

If so:  `sed -i s/type_/fs_type/g' unblob/handlers/filesystem/romfs.py`

Reply:
Sure!